### PR TITLE
media-sound/ocp: Version bump and general update of recipe

### PR DESCRIPTION
**Pull request by mywave82**
This depends on new version of app-arch/ancient #7793

### CODE REVIEW

**Comment by Begasus** [comment]
Needs https://github.com/haikuports/haikuports/pull/7793 to be merged before, so added wip for now.

File: media-sound/ocp/ocp-0.2.102.recipe [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+		--bindir=$commandBinDir \
+		--with-unifontdir-ttf=/system/data/fonts/ttfonts \
+		--with-unifontdir-otf=/system/data/fonts/otfonts
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	addAppDeskbarSymlink $binDir/ocp OpenCubicPlayer
```
Comment (Begasus) [inline media-sound/ocp/ocp-0.2.102.recipe:98]: This should be $commandBinDir :)
```diff
-	addAppDeskbarSymlink $binDir/ocp OpenCubicPlayer
+	addAppDeskbarSymlink $commandBinDir/ocp OpenCubicPlayer
```

Comment (mywave82) [reply media-sound/ocp/ocp-0.2.102.recipe]: This is now fixed

**Comment by mywave82** [comment]
> Maybe the version bump could be made by `git mv ocp-0.2.2.recipe ocp-0.2.102.recipe`, doing the necessary modifications to ocp-0.2.102.recipe, and `git add ocp-0.2.102.recipe` / `git commit` that.

Fixed

> PS: minor nit-pick/suggestion... PR title's "by tradition" are formatted like: `<package_name>: <what you did with the recipe>`, in this case, I would use something like: `ocp: update version to 0.2.102`

Fixed

File: media-sound/ocp/ocp-0.2.102.recipe (hunk 1 above)
Comment (Begasus) [reply media-sound/ocp/ocp-0.2.102.recipe]: Thanks!

**Comment by mywave82** [comment]
If there any more comments, do not be alarmed that I am not able to fix this before 5th of February due to a work trip :-D

**Comment by Begasus** [comment]
This one is fine by me, only needs libancient to be merged before we can merge this. :)

**Comment by mywave82** [comment]
> > > Maybe the version bump could be made by `git mv ocp-0.2.2.recipe ocp-0.2.102.recipe`, doing the necessary modifications to ocp-0.2.102.recipe, and `git add ocp-0.2.102.recipe` / `git commit` that.
> > 
> > 
> > Fixed
> 
> I'm still seeing only one commit adding the new recipe (instead of the suggested "git mv / modify / add / commit" cycle. Not that important, I guess, just mentioning it because maybe the "fix" got lost among other changes? git can be a PITA :-D

I will check again when I'm back home next week. I did git reset master, move out old new recipe, git mv, and overwrite recipe with new content before git commit. Strange strange :)

**Comment by mywave82** [comment]
> > > > Maybe the version bump could be made by `git mv ocp-0.2.2.recipe ocp-0.2.102.recipe`, doing the necessary modifications to ocp-0.2.102.recipe, and `git add ocp-0.2.102.recipe` / `git commit` that.
> > > 
> > > 
> > > Fixed
> > 
> > 
> > I'm still seeing only one commit adding the new recipe (instead of the suggested "git mv / modify / add / commit" cycle. Not that important, I guess, just mentioning it because maybe the "fix" got lost among other changes? git can be a PITA :-D 

I think I got it correct this time

**Comment by mywave82** [comment]
> Needs #7793 to be merged before, so added wip for now.

It has now been merged

**Comment by Begasus** [comment]
Thanks!

**Pull request merged**